Ark.Tools.FtpClient.Core/FtpConfig.cs

```diff
@@ -11,12 +11,12 @@ public class FtpConfig : IDisposable
     {
         private bool _isDisposed;
 
-        public Uri Uri { get; }
+        public Uri? Uri { get; }
         public NetworkCredential? Credentials { get; }
 
         public X509Certificate2? ClientCertificate { get; private set; }
 
-        public FtpConfig(Uri uri, NetworkCredential? credential = null, X509Certificate2? certificate = null)
+        public FtpConfig(Uri? uri = null, NetworkCredential? credential = null, X509Certificate2? certificate = null)
```

Comment:
this shall not change. As I previously stated, this PR shall not change any file in 'Core'